src/components/chips/js/contactChipsDirective.js

```diff
@@ -63,6 +65,8 @@ var MD_CONTACT_CHIPS_TEMPLATE = '\
               md-items="item in $mdContactChipsCtrl.queryContact($mdContactChipsCtrl.searchText)"\
               md-item-text="$mdContactChipsCtrl.itemName(item)"\
               md-no-cache="true"\
+              md-min-length="$mdContactChipsCtrl.minLength"\
+              md-has-not-found="true"\
```

Comment:
@DevVersion @topherfangio I removed the md-has-not-found attribute. This PR is dependent on merge of #9291.